src/core/manager.h

```diff
@@ -504,6 +504,12 @@ DEFINE_TRIVIAL_CLEANUP_FUNC(Manager*, manager_free);
 
 int manager_startup(Manager *m, FILE *serialization, FDSet *fds, const char *root);
 
+bool manager_dbus_is_running_full(Manager *m, bool deserialized);
+static inline bool manager_dbus_is_running(Manager *m) {
+        assert(m);
```

Comment:
Not a matter, but we usually do not add `assert()` such case.
(We add assertion e.g. when we access `m->foo` in inline function.)